packages/widget-chat/src/root.js

```diff
@@ -17,5 +17,3 @@ Root.propTypes = {
   accessToken: PropTypes.string.isRequired,
   userId: PropTypes.string.isRequired
 };
-
-export default injectWidgetLoader(Root);
```

Comment:
Yea, I removed the demo wrapper for now since that really shouldn't be a part of the CDN build.